CHANGELOG.md

```diff
@@ -1,8 +1,5 @@
-<<<<<<< HEAD
```

Comment:
uh oh we've got some merge conflict markers!

Reply:
oof, thanks! There were SO MANY, I tried to use GitHub and must have gotten lost 🙇🏽

Reply:
oops I also totally didn't realize you were _removing_ the markers! I think I just saw the markers and went "oh no!"